src/templates/pages/patterns/form-headings/index.js

```diff
@@ -0,0 +1,69 @@
+    <p class="push-bottom--double">Pages with more complex forms with multiple questions or multiple fieldsets should use the page heading to title the form.</p>
+
+    <h2 class="push-bottom--half plus-2 medium">Example - single question, single input</h2>
+    <iframe style="--height: 375px" class="example" title="Example form with single question" src={'/example/form-headings/form-headings-single'}></iframe>
+    <p class="push-bottom align-right"><a href="/example/form-headings/form-headings-single" rel="noopener" target="_blank">Open in a new tab</a></p>
+    <pre class="pre"><code class="code">{`${render(<CodeSingle />, null, { pretty: true })}`}</code></pre>
+    
+    <h2 class="push-bottom--half plus-2 medium">Example - single question, multiple choice</h2>
+    <iframe style="--height: 375px" class="example" title="Example form with multiple choice" src={'/example/form-headings/form-headings-single-multi'}></iframe>
+    <p class="push-bottom align-right"><a href="/example/form-headings/form-headings-single-multi" rel="noopener" target="_blank">Open in a new tab</a></p>
+    <pre class="pre"><code class="code">{`${render(<CodeSingleMulti />, null, { pretty: true })}`}</code></pre>
+
+    <h2 class="push-bottom--half plus-2 medium">Example - multiple questions</h2>
+    <iframe style="--height: 375px" class="example" title="Example form with multiple questions" src={'/example/form-headings/form-headings-multiple'}></iframe>
+    <p class="push-bottom align-right"><a href="/example/form-headings/form-headings-multiple" rel="noopener" target="_blank">Open in a new tab</a></p>
+    <pre class="pre"><code class="code">{`${render(<CodeMultiple />, null, { pretty: true })}`}</code></pre>
+
+    <h2 class="push-bottom plus-2 medium">Acceptance criteria</h2>
+    <p class="push-bottom--double">The following is a list of example acceptance criteria to test against when using this pattern. These critera should test that the specific markup requirements are met, and that the forms behave visually and functionally as expected.</p>
+
+    <h3 class="push-bottom--half plus-1 medium">For validation in developer tools / web inspector</h3>
+    <ul class="list list--tick push-bottom--double">
+        <li class="list-item">Only one <pre class="pre--inline">h1</pre> element should exist per page</li>
+        <li class="list-item">For form pages asking a single question with multiple choices, the <pre class="pre--inline">h1</pre> should be inside the <pre class="pre--inline">legend</pre> tag.</li>
+        <li class="list-item">For form pages asking a single question with a single input, the <pre class="pre--inline">h1</pre> tag should contain the appropriate <pre class="pre--inline">label</pre>.</li>
+        <li class="list-item">For form pages asking multiple questions, the <pre class="pre--inline">h1</pre> tag should be above all form inputs and summarise the entire form</li>
+    </ul>
```

Comment:
When you list the form types at the start, and through your examples, the order is : Single Q, Single with multiple inputs, Multiple Qs.  I think it would read more comfortably if the ACs listed the form types in the same order.